docs/user/gcp/install_upi.md

```diff
@@ -0,0 +1,507 @@
+resources of type 06_worker.py in the file.
+
+Copy the [template](../../../upi/gcp/06_worker.py) locally
+and create a resource definition file: 06_worker.yaml
+```
+imports:
+- path: 06_worker.py
+
+resources:
+- name: <name>
+  type: 06_worker.py
+  properties:
+    infra_id: '<infra_id>'
+    region: '<region>'
+
+    compute_subnet: '<compute_subnet>'
+    ignition: '<ignition>'
+    image_name: '<image_name'>
+    machine_type: '<machine_type>'
+    root_volume_size: '128'
+    service_account_email: '<service_account_email>'
+    zone: '<zone>'
+
+```
+- name: the name of the worker instance (ie worker-0)
+- infra_id: the infra_id from above (ie openshift-xxxx)
+- region: the region to deploy the cluster into (ie us-east1)
+- compute_subnet: the api path to the compute subnet
+  - `gcloud compute networks subnets describe ${INFRA_ID}-worker-subnet --format json | jq -r .selfLink`
+- ignition: the contents of the worker.ign file
+- image_name: the api path to the RHCOS image
+  - `gcloud compute images describe ${INFRA_ID}-rhcos-image --format json | jq -r .selfLink`
+- machine_type: The machine type of the instance (ie n1-standard-4)
+- service_account_email: the email address for the worker service account created above
+  - `gcloud iam service-accounts list | grep "^${INFRA_ID}-worker-node " | awk '{print $2}'`
+- zone: the zone for the worker node (ie us-east1-b)
+
+Create the deployment using gcloud.
+
+```
+gcloud deployment-manager deployments create ${INFRA_ID}-worker --config 06_worker.yaml
+```
+
+#### Approving the CSR requests for nodes
+
+The CSR requests for client and server certificates for nodes joining the cluster will need to be approved by the administrator.
+You can view them with:
+
+```console
+$ oc get csr
+NAME        AGE     REQUESTOR                                                                   CONDITION
+csr-8b2br   15m     system:serviceaccount:openshift-machine-config-operator:node-bootstrapper   Approved,Issued
+csr-8vnps   15m     system:serviceaccount:openshift-machine-config-operator:node-bootstrapper   Approved,Issued
+csr-b96j4   25s     system:node:ip-10-0-52-215.us-east-2.compute.internal                       Approved,Issued
+csr-bfd72   5m26s   system:node:ip-10-0-50-126.us-east-2.compute.internal                       Pending
+csr-c57lv   5m26s   system:node:ip-10-0-95-157.us-east-2.compute.internal                       Pending
+...
+```
+
+Administrators should carefully examine each CSR request and approve only the ones that belong to the nodes created by them.
+CSRs can be approved by name, for example:
+
+```sh
+oc adm certificate approve csr-bfd72
+```
+
+## Monitor for Cluster Completion
+
+```console
+$ openshift-install wait-for install-complete
+INFO Waiting up to 30m0s for the cluster to initialize...
+```
+
+Also, you can observe the running state of your cluster pods:
+
+```console
+$ oc get clusterversion
+
+$ oc get clusteroperators
```

Comment:
Including these without example output, and then following them with an output-included `oc get pods ...` feels strange.  If you want to only show output for some commands, probably do those first, and then have a separate section afterwards:

    The following monitoring commands may also be useful:

    ```sh
    oc get clusterversion
    ...
    ```

Reply:
Added output.